### DISCUSSION TITLE
Release of full MIMIC-IV as OMOP?

### DISCUSSION THREAD

**Discussion by ghost** [MIT-LCP/mimic-code#1581] [MIMIC-IV]
(Not sure whether this falls outside the range of this forum but I was not sure where else to ask)

Reading the [meeting notes](https://github.com/OHDSI/MIMIC/wiki/Meeting-minutes) of the group working on the MIMIC-to-OMOP conversion, it sounds like a release (on PhysioNet) of the full MIMIC-IV date set in the OMOP data model is approaching.

1. Is that indeed correct and, if so, will this be a release of the full MIMIC-IV dataset or merely an update of the existing [OMOP version of the 100-person MIMIC-IV demo set](https://physionet.org/content/mimic-iv-demo-omop/0.9/)?
2. If a release is underway, are there any estimates of when this release will be available on PhysioNet?

Thanks!

**Comment by alistairewj** [comment]
Not sure, @tompollard might know more. Probably relates to the many SSSOM PRs which are hanging in this repo!
